NewForce-Cohort-8/tabloidfullstack-suplex-syndicate
# Lv search by tag

### Description
Added ability for admins to deactivate users and prevent those users from logging in
[Trello Ticket 42](https://trello.com/c/GeuJq66r/42-42-search-by-tag)


### Testing Instructions
1. `git add` and `git commit` whatever you're working on in your branch
2. `git checkout main`
3. `git fetch -a`
4. `git checkout lv-search-by-tag`
5. If you don't see the most recent version of my files `git pull origin lv-search-by-tag`
6. 6. If you have not created the database please do so and follow Steps 7-8 otherwise proceed to Step 9
7. Run this [SQL script](https://github.com/NewForce-Cohort-8/tabloidfullstack-suplex-syndicate/blob/main/SQL/01_Tabloid_Create_DB.sql) to create the initial database
8. Run this [SQL script](https://github.com/NewForce-Cohort-8/tabloidfullstack-suplex-syndicate/blob/main/SQL/02_Tabloid_Seed_Data.sql) to seed the initial data
9. Login with `gteanby6@craigslist.orgx`
10. Select Posts
11. On the **grow frictionless e-tailers** post, add the Tags **C#** and **Javascript**
12. On the **productize turn-key convergence** post add the Tags **C#** and **Scary**
13. in the search by tag input field at the top of the post list, enter in **c**
14. The only two posts in the post list should be those titled above
15. clear the search by tag input field. 
16. The post list should now contain all posts again
17. In the search by tag input field at the top of the post list enter in **javascript**
18. The only post displayed should be the **grow frictionless e-tailers**
19. Clear the input field
20. The post list should be displaying all posts
21. In the search by tag input field at the top of the post list enter in **sc**
22. The only post displayed should be **productize turn-key convergence**
23. 
### Checklist
